### DISCUSSION TITLE
New Angular logo?

### DISCUSSION THREAD

**Discussion by pBouillon** [ng-icons/ng-icons#104] [Ideas]
Hey :wave:

The Angular team rebranded the logo during last year and major Angular libraries shifted to the new design already (RxAngular, NgRx, etc.). It could be great to update ng-icons' one as well!

**Comment by ashley-hunter** [comment]
Thanks for the suggestion! This certainly would be nice, I have updated the graphic on the readme but would like to also update the landing page, favicon and header image. I have spent some time in figma trying some ideas out, but nothing that I am 100% happy with yet.

If anyone has any ideas or suggestions - or wants to contribute a design, they would be very welcome! If not, I'm sure I'll come up with something after a bit more time.
